src/point.h

```diff
@@ -15,6 +15,11 @@ typedef struct
 	float8	lat;	/* latitude value in radians */
 } SPoint;
 
+/*
+ * For the correctness of the interface, the function returns the spoint itself
+*/
```

Comment:
Trivial: Please insert a space before the asterisk on this line. The asterisks should line up vertically.

Reply:
resolved